docs/multicluster/quick-start.md

```diff
@@ -0,0 +1,172 @@
+# Antrea Multi-cluster Quick Start
+
+In this quick start guide, we will set up a Antrea Multi-cluster ClusterSet with
+two clusters. One cluster will serve as both the leader and a member cluster,
+while another cluster will be a member only. The diagram belows shows the two
```

Comment:
@luolanzone : I feel we better to have a simple diagram to show the two clusters (with cluster ID), and ClusterSet (with leader and member cluster information).

Reply:
Sure, I haven't had the chance to draw the diagram yesterday, I will add it.

Reply:
A few comments for the diagram:

1. What you think we have a box for the ClusterSet that contains the two clusters? We can add ClusterSet ID there.
2. Add something like "leader" to the leader mc-controller to indicate it is the controller for leader.
3. And some way to indicate the Gateway Node too.
4. Could we add apiserver IP to Cluster A
5. Could we indicate Cluster A is "leader and member", and Cluster B is "member"